content/en/blog/2023-12-07-flux-release-around-the-corner/index.md

```diff
@@ -0,0 +1,24 @@
+---
+author: kingdonb
+date: 2023-12-07 00:00:00+00:00
+title: Flux release is just around the corner!
+description: "Flux v2.1.0: New monitoring setup with support for custom metrics, performance improvements and much more!"
+url: /blog/2023/12/flux-release-around-the-corner/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+Now that Flux has reached General Availability, there are only 3 releases per year. So, the maintainer team has been hard at work to create this Flux v2.2.0 release, which is slated to come out next week! Keep your eye out!
+
+While this next release addresses [hundreds of issues and has many improvements](https://github.com/fluxcd/flux2/issues/4410) that will be covered in more detail by a follow-up blog post, we want to highlight one crowning achievement of the next release: Flux helm-controller's reconciliation model has undergone a significant overhaul, this is the one we've all been waiting for!
+
+We hope many Flux users will rejoice in the improvements of the updated helm-controller, and that you'll all update to the latest release. This will help, in particular, with the issue of how the automatic recovery of releases could occasionally get stuck in a pending state. In addition, the latest release improves the observability of the release status by reflecting more detailed historic information in the status of the HelmRelease object.
+
+Lastly, the new release introduces the ability to enable drift detection on a per-object basis. This includes the option to ignore drift in specific fields using Kustomize-like target selectors and JSON pointer paths, which helps to reduce the noise and prevent upgrades from infinitely cycling. Helm Drift detection and correction also is no longer experimental since this release. Opt-in through `HelmRelease.spec.driftDetection` in the `v2beta2` API.
+
+There's much more in the upcoming Flux v2.2.0 release such as updates and improvements to Flux source-controller, image-reflector-controller, kustomize-controller, notification-controller, and the Flux CLI!
```

Comment:
We could just say "all controllers" or "all Flux components" instead of listing them all by name.

Reply:
@stefanprodan  Either way. I listed them because 1) new people might not know all of the controllers, so listing them could bring awareness and 2) not all controllers were listed in your release notes, so I thought to list these specifically (again as partial education). Either way, wdyt?

Reply:
Without providing further context about the enhancements to these specific controllers, or what they are responsible for. I feel this does not add much, as the names lack further context.

Also, paired with the helm-controller bit above. This names about any Flux component minus the image-automation-controller (which had updates as well, as Kustomize was updated there).